icu4c/source/common/brkeng.cpp

```diff
@@ -79,6 +79,7 @@ UnhandledEngine::findBreaks( UText *text,
                              int32_t /* startPos */,
                              int32_t endPos,
                              UVector32 &/*foundBreaks*/,
+                             UBool isPhraseBreaking,
```

Comment:
Change to 
                             UBool /* isPhraseBreaking */,
You only need to UBool in the function parameter but not the name isPhraseBreaking there. Same solution as for the startPos and foundBreaks 1 and 3 lines above.